### DISCUSSION TITLE
Error LIME

### DISCUSSION THREAD

**Discussion by AmriDomas** [teamalgoritma/community#745] [Q&A] [answered]
Dear Teamalgoritma,

Berikut error yang saya dapatkan jika menggunakan Decission tree

<img width="665" alt="image" src="https://github.com/teamalgoritma/community/assets/130373441/67ee2a54-f2d1-4f6c-b8c0-897720eaa891">

Apa solusinya?

<img width="675" alt="image" src="https://github.com/teamalgoritma/community/assets/130373441/2bf438fb-36ba-443a-8049-915500528b91">

Error pun sama dengan menggunakan Naive Bayes

Berbeda jika saya menggunakan Random Forest 

<img width="673" alt="image" src="https://github.com/teamalgoritma/community/assets/130373441/2ab69d9b-15fb-4109-a592-800eeacef218">

Tidak terdapat error

Regards,
Amri

**Comment by finesaaa** [comment]
Terima kasih pak @AmriDomas atas pertanyaannya, akan segera dijawab oleh tim kami. Mohon ditunggu

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear Pak @AmriDomas ,

Jika mengacu ke dokumentasi dari Lime, dijelaskan bahwa Limetidak support model dari decision tree dan naivebayes dan hanya support model-model dari library tertentu saja seperti caret, xgboost dan keras (dimana ini adalah model-model robust). [Model support Lime](https://rdrr.io/cran/lime/man/model_support.html).

Fungsi utama lime adalah ingin mengetahui prediktor apa saja yang paling berpengaruh bagi model-model robust (tidak interpretable). Namun, decision tree adalah model interpretable sehingga tidak diperlukan Lime untuk menginterpretasikan prediktor yang paling berpengaruh.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Reply by AmriDomas** [reply]
Terimakasih untuk sarannya

**Reply by IrfanChairurrachman** [reply]
Sama-sama Pak
